metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -117,6 +119,10 @@ def run(
     def run_pipeline_to_completion(
         pipeline: Pipeline, structured_report: Optional[str] = None
     ) -> int:
+        logger.info("Starting http server for Prometheus Python Client (Prometheus exporter)")
```

Comment:
not sure if that is reasonable - it will minorly affect startup time - do we really need to increase complexity just to decrease startup time for couple of seconds 
to be fair my preference is to keep it a bit simpler than a bit faster

Reply:
well, we need it everywhere for now, but still it does make sense to make it configurable

Reply:
> not sure if that is reasonable - it will minorly affect startup time - do we really need to increase complexity just to decrease startup time for couple of seconds to be fair my preference is to keep it a bit simpler than a bit faster

but you're waiting for a minute after ingestion ends - that's the main reason of this option

Reply:
ah  - right - that one is important part - makes sense